ArcGISRuntimeSDKQt_CppSamples_Widgets/Maps/ChangeBasemap/ChangeBasemap.cpp

```diff
@@ -36,13 +36,10 @@ ChangeBasemap::ChangeBasemap(QWidget* parent) :
     m_basemapCombo = new QComboBox(this);
     m_basemapCombo->adjustSize();
     m_basemapCombo->setStyleSheet("QComboBox#combo {color: black; background-color:#000000;}");
-    m_basemapCombo->addItems(QStringList() << "Topographic"
-                             << "Streets"
-                             << "Imagery"
-                             << "Oceans");
+    m_basemapCombo->addItems(QStringList { "Topographic", "Streets", "Imagery", "Oceans" });
 
     // Connect the combo box signal to lambda for setting new basemaps
-    connect(m_basemapCombo, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), [=](int i) {
+    connect(m_basemapCombo, QOverload<int>::of(&QComboBox::currentIndexChanged), [this](int i) {
```

Comment:
👍

Reply:
This is nice @anmacdonald. The previous syntax is what the Qt Company recommends in their doc with older versions of Qt. `QOverload` is much nicer.